packages/stencil/components/pagination/src/components/osds-pagination/osds-pagination.tsx

```diff
@@ -172,36 +177,43 @@ export class OsdsPagination implements OdsPagination<OdsStencilMethods<OdsPagina
 
   renderArrows(direction: 'left' | 'right') {
     const { disabled } = this;
-    const arrowIcon = direction === 'left' ? OdsIconName.CHEVRON_LEFT : OdsIconName.CHEVRON_RIGHT;
+    const isLeft = direction === 'left';
+    const arrowIcon = isLeft ? OdsIconName.CHEVRON_LEFT : OdsIconName.CHEVRON_RIGHT;
 
     return (
       <li class="arrows">
-        <osds-button
-          variant={OdsButtonVariant.ghost}
-          color={OdsThemeColorIntent.primary}
-          disabled={disabled || (direction === 'left' && this.current === 1) || (direction === 'right' && this.current >= this.pageList.length)}
-          onKeyDown={(event: KeyboardEvent) => {
-            if (direction === 'left') {
-              this.handlePreviousKeyDown(event, Number(this.current));
-            } else {
-              this.handleNextKeyDown(event, Number(this.current));
-            }
-          }}
-          onClick={() => {
-            if (direction === 'left') {
-              this.handlePreviousClick(Number(this.current));
-            } else {
-              this.handleNextClick(Number(this.current));
-            }
-          }}
-          size={OdsButtonSize.sm}>
-          <osds-icon
-            size={OdsIconSize.sm}
-            name={arrowIcon}
+        <osds-tooltip>
+          <osds-tooltip-content slot="tooltip-content">
+            { isLeft ? this.labelTooltipPrevious : this.labelTooltipNext }
```

Comment:
did you try using `key` instead to trigger rerender on change?
`forceUpdate` on the whole element is quite costly, would be nice to avoid using it